CMakeLists.txt

```diff
@@ -3,12 +3,13 @@ cmake_minimum_required(VERSION 3.16)
 
 # Include the "single source of truth" for the clang-format version
 include(clang-format_version.cmake)
+string(REPLACE "-" "" CLANG_FORMAT_VERSION_SHORT "${CLANG_FORMAT_VERSION}")
```

Comment:
The reason for this line is to handle rcN releases such as: `https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0-rc4/llvm-13.0.0rc4.src.tar.xz`

The main release tag has `13.0.0-rc4`, but the name of the downloaded file uses `13.0.0rc4` (weird).

Reply:
Thanks for explaining that line, it was indeed a bit odd when reading through the changes.